MANIFEST.md

```diff
@@ -0,0 +1,16 @@
+Manifest 
+---------
+
+Celem naszych spotkań jest poszerzanie wiedzy i wspólne spędzanie czasu w przyjaznej atmosferze. W związku z tym oczekujemy, że wszyscy bez wyjątku będą darzyć się wzajemnym szacunkiem.
+
+Jako społeczność oczekujemy, by każdy z nas stosował się do przedstawionych poniżej zasad, ponieważ uważamy, że pozwoli to wszystkim komfortowo i bezstresowo uczestniczyć w życiu naszej grupy:
+
+* wroc.py jako miejsce przyjazne dla wszystkich pozostaje przestrzenią wolną od formułowania publicznych ocen w kwestiach światopoglądowych, zwłaszcza tych związanych z płcią, preferencjami seksualnymi, budową ciała, rasą lub wyznaniem.
+
+* W publicznej dyskusji, jak również w prywatnych wypowiedzi staramy się zaniechać agresywnych bądź rubasznych wypowiedzi wprowadzających celowe dwuznaczności, mogące zostać uznane przez inne osoby za wulgarne czy niestosowne. Podobne przypadki mogą powodować złe samopoczucie u części uczestników i nie wnoszą niczego wartościowego do naszych spotkań.
+
+* Jeśli w pewnych kwestiach nie mamy pewności, gdzie przebiega granica komfortu, to warto zdać się na zdrowy rozsądek.
+
+* Osoby niereformowalne i uprzykrzające życie innym będą wypraszane z naszych spotkań, a w szczególnych przypadkach wykluczane z dalszego udziału w życiu społeczności.
```

Comment:
caly ten manifest sie sprowadza do jednego zdania "jestes niegrzeczny to zrobimy ci kuku". mowisz, ze kuku mozecie zrobic i tak niezaleznie od tego manifestu. wychodzi na to ze ten manifest jest zbedny, bo z takimi sytuacjami mozna sobie radzic jak normalni ludzie - bez zadnych regulaminow.